shibboleth/resources/prod/shibboleth2.xml

```diff
@@ -0,0 +1,160 @@
+<SPConfig xmlns="urn:mace:shibboleth:3.0:native:sp:config"
+          xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
+          clockSkew="180">
+
+    <OutOfProcess tranLogFormat="%u|%s|%IDP|%i|%ac|%t|%attr|%n|%b|%E|%S|%SS|%L|%UA|%a"/>
+
+
+    <!-- The ApplicationDefaults element is where most of Shibboleth's SAML bits are defined. -->
+    <ApplicationDefaults id="default"
+                         policyId="default"
+                         entityID="https://edview.unilu.ch/shibboleth"
+                         homeURL="https://edview.unilu.ch/home"
```

Comment:
This also seems to be outdated. The "/home" url doesnt exist anymore. I have already pushed this change to the repo.